python_modules/dagster/dagster/_grpc/client.py

```diff
@@ -189,12 +189,12 @@ def _streaming_query(
 
     def ping(self, echo: str):
         check.str_param(echo, "echo")
-        res = self._query("Ping", api_pb2.PingRequest, echo=echo)
+        res = self._query("Ping", api_pb2.PingRequest, echo=echo)  # type: ignore
```

Comment:
what exactly are the type errors being suppressed here?

Reply:
bump

Reply:
See below

Reply:
brutal - wonder what else we can do to manage this better - maybe our own layer of indirection if the mentioned external libraries are not usable.